.gitlab-ci.yml

```diff
@@ -21,84 +21,112 @@ generate-scripts:
       - install_script_agent6.sh
       - install_script_agent7.sh
 
-test-opensuse-13:
+.test:
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/docker_x64:$DATADOG_AGENT_BUILDIMAGES
   tags: ["runner:docker"]
   stage: test
   dependencies: ["generate-scripts"]
+  script:
+    - ./test/dockertest.sh --image "registry.ddbuild.io/images/mirror/${IMAGE}" --script install_script_agent6.sh 
+      --minor-version "${MINOR_VERSION}"
+      --expected-minor-version "${EXPECTED_MINOR_VERSION}" 
+      --flavor "${FLAVOR}" 
+      --old-suse "${OLD_SUSE}" 
+      --apt-url "$APT_URL" 
+      --apt-repo-version "$APT_REPO_VERSION_AGENT6" 
+      --yum-url "$YUM_URL" 
+      --yum-version-path "$YUM_VERSION_PATH_AGENT6" 
+    - ./test/dockertest.sh --image "registry.ddbuild.io/images/mirror/${IMAGE}" --script install_script_agent7.sh
+      --minor-version "${MINOR_VERSION}"
+      --expected-minor-version "${EXPECTED_MINOR_VERSION}"
+      --flavor "${FLAVOR}"
+      --old-suse "${OLD_SUSE}"
+      --apt-url "$APT_URL"
+      --apt-repo-version "$APT_REPO_VERSION_AGENT7"
+      --yum-url "$YUM_URL"
+      --yum-version-path "$YUM_VERSION_PATH_AGENT7"
+  variables:
+    OLD_SUSE: "false"
```

Comment:
question ❓ 
if you set it by default there are 2 definition of the variable in the `test-opensuse-13` case. Is there a risk the definition not to be done in the correct order (and resulting on OLD_SUSE=false in the test so not testing what we want silently)?

Reply:
I think it is safe, when you extend a job, extended job fields are overriden by the fields defined in the jobs with `extends:` attribute. So here all jobs extending `.test` job will have a variable `OLD_SUSE: false` unless we explicitly specify another value for this variable